### PR TITLE
feat: export loadConfigFile type

### LINKED ISSUE #4841: Missing typings for rollup/loadConfigFile
### Expected Behavior / Situation

Typings for [`rollup/loadConfigFile`](https://rollupjs.org/javascript-api/#programmatically-loading-a-config-file) should be exported.

See also: https://github.com/rollup/rollup/issues/4660#issuecomment-1274342275

### Actual Behavior / Situation

No typings for `rollup/loadConfigFile` are exported.

### Modification Proposal

Export typings for `rollup/loadConfigFile`.

### PR DESCRIPTION
**Pull request by TrickyPi**
This PR contains:

- [ ] bugfix
- [x] feature
- [ ] refactor
- [ ] documentation
- [ ] other

Are tests included?

- [ ] yes (_bugfixes and features will not be merged without tests_)
- [x] no

Breaking Changes?

- [ ] yes (_breaking changes will not be merged unless absolutely necessary_)
- [x] no

List any relevant issue numbers:
resolves #4841

### CODE REVIEW

File: build-plugins/copy-types.ts [hunk 1]
```diff
@@ -16,3 +16,24 @@ export default function copyTypes(fileName: string): Plugin {
 		name: 'copy-types'
 	};
 }
+
```
Comment (lukastaegert) [inline build-plugins/copy-types.ts:19]: As `copyTypes` is now only used for the browser types, maybe rename it to `copyBrowserTypes` and remove/hard code the `fileName` parameter to `rollup.browser.d.ts`.

Then `copyAllTypes` could be renamed `copyNodeTypes`

**Review by lukastaegert — COMMENTED** [review]
Awesome, that is exactly how I thought it should be done! One minor comment with regard to naming the build plugins, and I will need to do a little more testing to see if the types work correctly in all cases (but I trust they do).

File: build-plugins/copy-types.ts (hunk 1 above)
Comment (TrickyPi) [reply build-plugins/copy-types.ts:19]: Ok, will do.

Comment (TrickyPi) [reply build-plugins/copy-types.ts:19]: I changed the name of `copyTypes` to `copyRollupType` and added a new `copyBrowserTypes ` function.

**Review by lukastaegert — APPROVED** [review]
Looks great, thanks a lot!

**Pull request merged**